go.php

```diff
@@ -82,13 +82,49 @@
 
 header('HTTP/1.1 302 Moved Temporarily');
 if (array_key_exists($from, $mapping)) {
-    header('Location: ' . $location . $mapping[$from]);
+    $subPath = $mapping[$from];
 } else {
     if (strpos($from, 'admin-') === 0) {
```

Comment:
```suggestion
} elseif (strpos($from, 'admin-') === 0) {
```

And `s/else if/elseif/g` on the lines below. I’m sorry about the unrelated review but this kind of things trigger me.